features.md

```diff
@@ -160,3 +160,78 @@ Known limitations
 > editors, e.g. VSCode, provide a way to preview the rename changes before
 > applying them; while some just apply the changes directly.
 {:.tip}
+
+## Semantic Highlighting
+
+clangd computes semantic highlight tokens to enhance the source code theming. {.v9}
+
+![screenshot: highlight](screenshots/semantic_highlight.png)
+
+Tokens may be associated with one highlight kind and multiple highlight modifiers, as defined by the following section.
+
+### Kinds
+
+clangd supports following standard LSP token kinds:
+
+| Kind                   | LSP Name      | Note                       
+|------------------------|---------------|--------------------------
+| Variable{.v9}          | variable      |                          
```

Comment:
I don't think we need separate columns for clangd's internal kind and the LSP kind. The internal kind is not exposed to the client in any way.

Let's just combine them into one column, we can call it `LSP Kind`. (And e.g. `variable` only needs one row.)

Reply:
clangd's internal kinds are needed for https://reviews.llvm.org/D148489, where we use that as configuration items.

Reply:
Ah, good point! When reviewing https://reviews.llvm.org/D148489, I didn't fully realize the implication that we were exposing the internal kinds via this config mechanism. In retrospect, it would probably have been better to have the config keys be the LSP names, but it's too late for that now. (We can still consider a future change to make the internal kinds be 1:1 with the LSP kinds.)